doc/building_windows.md

```diff
@@ -0,0 +1,213 @@
+# Building Julius for Windows
+
+You have two main options in how you want to build Julius. You can either:
+
+* [Build using **Microsoft Visual Studio**](#a-using-microsoft-visual-studio-64-bit-build); or
+* [Install CMake and MinGW and **use the command line**](#b-using-cmake-and-mingw-via-command-line).
+
+
+## A. Using Microsoft Visual Studio (64-bit build)
+
+1. Make sure you have [Microsoft Visual Studio](https://visualstudio.microsoft.com/vs/) installed. Please note that only Visual Studio 2017 (fully updated) or Visual Studio 2019 can be directly used.
+These instructions assume you are using Visual Studio 2019.
+
+2. Download the `SDL2` and `SDL2_mixer` development libraries, both as a zip file:
+
+	* [SDL2 dev libraries for Visual Studio](https://www.libsdl.org/release/SDL2-devel-2.0.10-VC.zip);
+	* [SDL2_mixer dev libraries for Visual Studio](https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-devel-2.0.4-VC.zip).
+
+3. Extract the `SDL2` and `SDL2_mixer` development libraries to an easy to remember folder (I recommend something like `C:\libs`).
+
+4. Open Microsoft Visual Studio. Select `Clone or check out code`.
+ 
+ 	![](images/building_windows_msvc_1.jpg)
```

Comment:
Can we use non-HD screenshots, if they are necessary at all? 2048x1420px is overkill. Also, indexed PNGs should be smaller than JPGs.

Reply:
Sure!